tests/integration/test_fim/test_registry/test_ignore/test_ignore_registry.py

```diff
@@ -94,17 +81,103 @@ def test_ignore_registry(key_string, key_object, tags_to_apply, get_configuratio
         Run test if matches with a configuration identifier, skip otherwise.
     """
     check_apply_test(tags_to_apply, get_configuration['tags'])
+    scheduled = get_configuration['metadata']['fim_mode'] == 'scheduled'
+    # Create registry
+    create_registry(registry_parser[root_key], os.path.join(registry, subkey), arch)
+    # Go ahead in time to let syscheck perform a new scan
+    check_time_travel(scheduled, monitor=wazuh_log_monitor)
+
+    if triggers_event:
+        event = wazuh_log_monitor.start(timeout=global_parameters.default_timeout,
+                                    callback=callback_detect_event,
+                                    error_message='Did not receive expected '
+                                                    '"Sending FIM event: ..." event',
+                                    accum_results=2).result()
+
+        assert event[0]['data']['type'] == 'added', f'Wrong event type.'
+        assert event[0]['data']['path'] == os.path.join(root_key, registry, subkey), f'Wrong key path.'
+        assert event[0]['data']['arch'] == '[x32]' if arch == KEY_WOW64_32KEY else '[x64]', f'wrong key arch.'
+
+        assert event[1]['data']['type'] == 'modified', f'Parent key event type not equal'
+        assert event[1]['data']['path'] == os.path.join(root_key, registry), f'Wrong parent key path.'
+        assert event[1]['data']['arch'] == '[x32]' if arch == KEY_WOW64_32KEY else '[x64]', f'Parent key arch not equal.'
+    else:
+        # The ignore event is generated before the event of the parent key.
+        while True: # Look for the ignore event of the created key
```

Comment:
Please, have at least two spaces before an inline comment: https://www.flake8rules.com/rules/E261.html
```suggestion
        while True:  # Look for the ignore event of the created key
```